Docker image to run batch jobs

ttung opened the pull request:
### Description
This docker image is based on nextstrain/base and sets up a virtualenv with aspen's dependencies installed.  This can be used to invoke a variety of workflows.

#### Issue
[ch126778](https://app.clubhouse.io/genepi/story/126778)

### Test plan
docker run -it --rm aspen/batch:latest trunk ls

phoenixAja reviewed (approved):
so will we be using this container in the nextstrain job definition then?

ttung commented:
> so will we be using this container in the nextstrain job definition then?

No, this is the aspen batch job image.

Commit:
Docker image to run batch jobs

### Description
This docker image is based on nextstrain/base and sets up a virtualenv with aspen's dependencies installed.  This can be used to invoke a variety of workflows.

#### Issue
[ch126778](https://app.clubhouse.io/genepi/story/126778)

### Test plan
docker run -it --rm aspen/batch:latest trunk ls
